packages/whosmysanta-backend/package.json

```diff
@@ -1,48 +1,21 @@
 {
   "name": "whosmysanta-backend",
   "version": "1.0.0",
-  "description": "GraphQL server for WhosMySanta",
-  "private": true,
-  "author": "WhosMySanta <hello@whosmysanta.com>",
+  "main": "index.js",
```

Comment:
Would we want to publish this to npm in the future? Imho, this is an app (express with graphql) rather than an npm package. Unlike `whosmysanta-core` and `whosmysanta-js`, we may publish in the future.
`index.js` in the project root doesn't exist anyway.